app/views/discord_engine/interactions/connect.jbuilder

```diff
@@ -1 +1,6 @@
-json.partial! 'discord_engine/partials/single_line', resolver: @resolver
+json.partial!(
+  'discord_engine/partials/single_line',
+  type: @resolver.callback.type,
+  flags: @resolver.respond_to?(:flags) ? @resolver.flags : nil,
```

Comment:
https://apidock.com/rails/Object/try

```suggestion
  flags: @resolver.try(:flags),
```

Reply:
Also, this serializer is tied to the `Connect` resolver, right? why checking if `flags` exist? Isn't it clear that it doesn't because the resolver is not defining the attribute?

```suggestion
  flags: nil,
```

I mean, if you were to build this serializer to be reusable in different scenarios and some of them might have `flags` then this makes sense to me. But this is very straightforward, it will always be tied to the `Connect` resolver, right?